tests/dummy/app/templates/docs/components/au-card.md

```diff
@@ -149,7 +149,7 @@
 
 {{#docs-demo as |demo|}}
```

Comment:
I think it should be added to the argument table at the bottom of the docs page as well?